doc/user_guide/electron_holography.rst

```diff
@@ -0,0 +1,72 @@
+Electron Holography
+*******************
+
+HyperSpy provides the user with two classes which can be used to process electron microscopy data:
+ 

```

Comment:
I definitely intended to write "electron holography", thanks for pointing that out :-)!